tests/functional/AdminBlockCest.php

```diff
@@ -675,7 +675,8 @@ public function ForceDeleteBlock(FunctionalTester $I)
             ],
             $user
         );
-        $I->sendDelete($this->url . '/' . $block->id);
+        // Since we can delete block even it wasn't soft deleted we don't need it in tests
+        //$I->sendDelete($this->url . '/' . $block->id);
```

Comment:
By this change in test I need to modify method in CMS, https://github.com/GrupaZero/cms/pull/82.